lib/hocon/impl/parseable.rb

```diff
@@ -101,7 +101,11 @@ def parse
     self.class.force_parsed_to_object(parse_value(options))
   end
 
-  def parse_value(base_options)
+  def parse_value(base_options = nil)
+    if base_options.nil?
+      base_options = options
+    end
+
```

Comment:
I can never remember the language-specific rules for when those defaults get evaluated.  In some languages the default only gets evaluated once, when the code is loaded, and that can cause weird behaviors, so I generally try to just avoid putting method calls as defaults in the args list just to be safe.